projects/fundamental-ngx/src/lib/action-bar/action-bar-description.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ActionBarDescriptionComponent } from './action-bar-description.component';
+
+describe('ActionBarDescriptionComponent', () => {
+    let component: ActionBarDescriptionComponent;
+    let fixture: ComponentFixture<ActionBarDescriptionComponent>;
+
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+            declarations: [ActionBarDescriptionComponent]
+        }).compileComponents();
+    }));
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(ActionBarDescriptionComponent);
+        component = fixture.componentInstance;
+        fixture.detectChanges();
+    });
+
+    it('should create', () => {
+        expect(component).toBeTruthy();
```

Comment:
does this bring any value(beside code coverage)?

Reply:
The "expect(component).toBeTruthy();" test is created when automatically when you generate a component using the angular CLI.  In addition to providing code coverage I think this test helps prevent accidental renaming/moving/deletion of components